### ISSUE TITLE
Feature request: Focus should move with the tab

### ISSUE THREAD

**Issue by Mellbourn** [anka-213/chrome-moveTab#1]
I think the focus should be moved to the tab that was moved, not remain on the current window.

Motivation: This behaviour would be much more intuitive, the tab you are moving is the one that has focus and is often the one you care about currently.

If you don't feel like working more on this extension, I may take a stab at making a fork and do this myself.

**Comment by anka-213** [comment]
Good point. I did have the current behaviour intentionally because it makes it easier to move multiple tabs at once, but there are arguments for both behaviours.

I'm not currently using this extension myself, so a fork is probably the best option for now.

**Comment by Mellbourn** [comment]
Hey again! Just to let you know, I did make a fork: https://github.com/Mellbourn/ShiftTab and published it https://chrome.google.com/webstore/detail/shift-tab/bjiiinamdgebpdcfbobnjnapnnndejnh

**Comment by anka-213** [comment]
nice!
